package.json

```diff
@@ -89,7 +89,8 @@
     "**/nth-check": "^2.0.1",
     "**/qs": "^6.10.3",
     "**/trim": "^0.0.3",
-    "**/typescript": "4.0.2"
+    "**/typescript": "4.0.2",
+    "**/axios": "^0.27.2"
```

Comment:
This list should be alphabetized. Typically yarn does this automatically but I don't believe it works for `resolutions`.

Reply:
Changing